helpers/buildExports.js

```diff
@@ -59,38 +62,72 @@ function writePkg(pkgPath, content) {
 }
 
 /**
- * Insert an array of icons into the preconstruct entrypoint array of a package.json
+ * Insert an array of component svgs into the preconstruct entrypoint array of a package.json
  *
- * @param  {array}  icons   - An array of icons
+ * @param  {array}  svgs   - An array of component svgs
  * @param  {string} pkgPath - The path to the package.json file
  */
-function insertPkg(icons, pkgPath) {
+function insertPkg(svgs, pkgPath) {
 	pkgPath = path.normalize(`${process.cwd()}/${pkgPath}`);
 	const pkg = require(pkgPath);
 
-	pkg.preconstruct.entrypoints = ['.', ...icons];
+	if (!pkg.preconstruct) {
+		pkg.preconstruct = {};
+	}
+
+	pkg.preconstruct.entrypoints = ['.', ...svgs];
 
 	writePkg(pkgPath, pkg);
 
 	console.info(chalk.green('✅ package.json file written successfully'));
 }
 
 /**
- * Fix the source key inside each icons package.json file
+ * Fix the source key inside each svgs package.json file
  *
- * @param  {array} icons - An array of all icons
+ * @param  {array} svgs - An array of all component svgs
  */
-function fixSource(icons) {
-	icons.map(icon => {
-		const pkgPath = path.normalize(`${process.cwd()}/${icon}/package.json`);
-		const pkg = require(pkgPath);
-
-		if (!pkg.preconstruct) {
-			pkg.preconstruct = {};
+function fixSource(svgs, component) {
+	svgs.map(svg => {
+		// check if package exists
+		const folderPath = path.normalize(`${process.cwd()}/${svg}`);
+		if (!fs.existsSync(folderPath)) {
+			// have a log about creating folder here
+			// need to have error handlers here
+			try {
+				fs.mkdirSync(folderPath);
```

Comment:
All of this is done inside `preconstruct` and wasn't needed here :)

Reply:
It wasn't creating folders for me before thats why I added it in. Ill check with new version

Reply:
Yeah that's why I said we need to run it twice. It does create it even in the old version but you need to run it first.

Reply:
I removed this part, removed one of the symbol folders and ran the build script and it errors out like before
```
Error: Cannot find module '/Users/Jeremy/Dev/gel-symbol/components/symbol/AmericanExpressSymbol/package.json'
```

Reply:
Did yoiu run `preconstruct`?

Reply:
ahhh I was always running preconstruct dev not build. All good I'll remove it.

Reply:
sorry that's my fault for not communicating properly... 😁